modules/renter/downloadchunk.go

```diff
@@ -166,6 +171,24 @@ func (udc *unfinishedDownloadChunk) returnMemory() {
 	}
 }
 
+// addChunkToCache adds the chunk to the cache if the download is a streaming
+// endpoint download.
+func (udc *unfinishedDownloadChunk) addChunkToCache(data []byte) {
+	// TODO is it safe to read the type without a lock?
```

Comment:
naming convention is to call it 'static' if the field doesn't change. If you want to read it without holding a lock, you'll need to change it to static, then go through the code and check every place the field is used and make sure that it only ever changes at init.